File: SortStringsAlphabetically.py
# Sort Strings Alphabetically
# - Given a list of strings `words`, sort them in alphabetical order.
# - Constraints:
#   - 1 <= len(words) <= 10^4

from typing import List

class Solution:
    def sortAlphabetically(self, words: List[str]) -> List[str]:
        return sorted(words)

File: SortStringsAlphabetically.cs
// Sort Strings Alphabetically
// - Given a list of strings, return the list sorted in alphabetical order.
// - Time: O(n log n), Space: O(n)

using System;
using System.Linq;

public class Solution
{
    public string[] SortStringsAlphabetically(string[] words)
    {
        return words.OrderBy(w => w).ToArray();
    }
}
